Assets/MXR.SDK/Runtime/Types/HomeScreenTypes.cs

```diff
@@ -38,6 +38,20 @@ public enum HomeScreenView {
         PASSCODE
     }
 
+    /// <summary>
+    /// The different language types recognised by the SDK. 
+    /// </summary>
+    [Serializable]
+    public enum HomeScreenDisplayLanguage {
```

Comment:
nit: Can we change this to `DisplayLanguage` and move it inside the `CustomLauncherSettings` class (becoming `CustomLauncherSettings.DisplayLanguage`? Doing so will self document the scope and relevant use case of this class.

Reply:
Addressed.